src/openforms/utils/json_logic/datastructures.py

```diff
@@ -54,6 +56,11 @@ def get_input_components(
 
             # TODO: this *may* be nested var.var expressions
             key = cast(str, node.arguments[0])
+            if key not in components_map:
+                inputs.append(
+                    InputVar(key=key, value=context[key], step_name="", label="")
```

Comment:
I think this fails for selectboxes components. For example, if you have a selectboxes component with values A and B, the value of this component in the data is `{"a": False, "b": False}`. In the logic rule you would have the trigger contain `{"var": "selectBoxes.a"}`, but as a key `"selectBoxes.a"` doesn't correspond to any component.